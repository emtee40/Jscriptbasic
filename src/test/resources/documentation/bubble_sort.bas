a[0] = 3
a[1] = 2
a[2] = 7
a[3] = -1
a[4] = 58
a[5] = 0
a[6] = 99

for i=0 to length(a)-1
for j=0 to length(a)-1
 if a[i] < a[j] then
   x = a[i]
   a[i] = a[j]
   a[j] = x
 endif
next j
next i

for i=0 to length(a)-1
  print a[i],"\n"
next i