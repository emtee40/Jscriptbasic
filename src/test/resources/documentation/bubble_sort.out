-1
0
2
3
7
58
99
